packages/graphback-codegen-schema/tests/mock.graphql

```diff
@@ -48,10 +48,13 @@ See issue https://github.com/aerogear/graphback/issues/1299
 type  Issue {
   id: ID!
   category: IssueCategory
+  createdAt: DateTime
```

Comment:
CC @ssd71 We going to enable scalar support in this PR so your hacks might be swapped later. However we will still need to supply set of the `graphback supported` scalars that we will require for both core and datasync in separate PR so please bear with us until we get this sorted.